### PR TITLE
feat(comment): basic angular wrapper implementation

### PR DESCRIPTION
**Pull request by Supamiu**
## Description

The comment component is used to display comments or discussions on blocks

Each block is composed of informations about the author (`avatar`, a `TemplateRef` and `authorName`, a `PortalContent` (`string | TemplateRef`)) and handles comments that are placed inside of it as child `lu-comment` components.

-----

-----

### CODE REVIEW

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/18438/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2866/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/22155/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2866/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/22216/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2866/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2866/storybook

File: packages/ng/comment/comment-chat/comment-chat.component.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<ol class="commentWrapperChat">
```
Comment (jeremie-lucca) [inline packages/ng/comment/comment-chat/comment-chat.component.html:1]: I wonder if this shouldn't be considered as a new component as I heard Clément speaking about "Chatbox". I'll investigate.

File: packages/scss/src/components/comment/component.scss [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
+@use '@lucca-front/scss/src/components/avatar/exports' as avatar;
+@use '@lucca-front/scss/src/commons/utils/reset';
 
 @mixin component($atRoot: namespace.$defaultAtRoot) {
 	container: comment / inline-size;
 	display: flex;
 	flex-direction: column;
-	gap: var(--pr-t-spacings-50);
-	max-width: 40rem;
+	gap: var(--pr-t-spacings-100);
```
Comment (jeremie-lucca) [inline packages/scss/src/components/comment/component.scss:9]:
```suggestion
	gap: var(--pr-t-spacings-50);
```

File: packages/scss/src/components/comment/component.scss [hunk 3]
```diff
@@ -73,26 +95,18 @@
...
+	@include reset.list;
 
-		.commentWrapper-item {
-			list-style: none;
+	display: flex;
+	flex-direction: column;
+	gap: var(--pr-t-spacings-200);
 
-			&.mod-WrapperAvatar ~ &:not(.mod-WrapperAvatar) {
-				padding-left: var(--components-comment-content-margin);
```
Comment (jeremie-lucca) [inline packages/scss/src/components/comment/component.scss:94]: Is there a reason behind this deletion? 
It seems to break the comment's wrap (with avatar only on first comment) :
image

**Review by jeremie-lucca — CHANGES_REQUESTED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2866/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/23622/ui-diff)

**Comment by jeremie-lucca** [comment]
(Waiting design feedback on components naming before mergin')

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2866/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2866/storybook

**Review by jeremie-lucca — DISMISSED** [review]

**Review by jeremie-lucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2866/storybook

**Pull request merged**